mlflow/johnsnowlabs.py

```diff
@@ -0,0 +1,874 @@
+                      environment::
+
+                        {
+                            'name': 'mlflow-env',
+                            'channels': ['defaults'],
+                            'dependencies': [
+                                'python=3.8.15',
+                                'pyspark=2.3.0'
+                            ]
+                        }
+    :param dfs_tmpdir: Temporary directory path on Distributed (Hadoop) File System (DFS) or local
+                       filesystem if running in local mode. The model is be written in this
+                       destination and then copied to the requested local path. This is necessary
+                       as Spark ML models read from and write to DFS if running on a cluster. All
+                       temporary files created on the DFS are removed if this operation
+                       completes successfully. Defaults to ``/tmp/mlflow``.
+    :param sample_input: A sample input that is used to add the MLeap flavor to the model.
+                         This must be a PySpark DataFrame that the model can evaluate. If
+                         ``sample_input`` is ``None``, the MLeap flavor is not added.
+
+    :param signature: :py:class:`ModelSignature <mlflow.models.ModelSignature>`
+                      describes model input and output :py:class:`Schema <mlflow.types.Schema>`.
+                      The model signature can be :py:func:`inferred <mlflow.models.infer_signature>`
+                      from datasets with valid model input (e.g. the training dataset with target
+                      column omitted) and valid model output (e.g. model predictions generated on
+                      the training dataset), for example:
+
+                      .. code-block:: python
+
+                        from mlflow.models.signature import infer_signature
+
+                        train = df.drop_column("target_label")
+                        predictions = ...  # compute model predictions
+                        signature = infer_signature(train, predictions)
+    :param input_example: Input example provides one or several instances of valid
+                          model input. The example can be used as a hint of what data to feed the
+                          model. The given example will be converted to a Pandas DataFrame and then
+                          serialized to json using the Pandas split-oriented format. Bytes are
+                          base64-encoded.
+    :param pip_requirements: {{ pip_requirements }}
+    :param extra_pip_requirements: {{ extra_pip_requirements }}
+    :param metadata: Custom metadata dictionary passed to the model and stored in the MLmodel file.
+
+                     .. Note:: Experimental: This parameter may change or be removed in a future
+                                             release without warning.
+
+    .. code-block:: python
+        :caption: Example
```

Comment:
please update this example to be flavor-relevant

Reply:
updated thanks